src/state/quick-bar-mixin.ts

```diff
@@ -48,9 +48,9 @@ export default <T extends Constructor<HassElement>>(superClass: T) =>
 
     private _registerShortcut() {
       tinykeys(window, {
-        KeyE: (ev) => this._showQuickBar(ev),
-        KeyC: (ev) => this._showQuickBar(ev, true),
-        KeyM: (ev) => this._createMyLink(ev),
+        e: (ev) => this._showQuickBar(ev),
```

Comment:
@bramkragten @Salamandar apologies I didn't receive notifications from github.
Salamander is right. However if we go back to using the key we have a problem. This will probably be solved by integrating Keyboard.getLayoutMap() in to tinykeys but this is still experimental in the browsers so it will take a long time.

The only viable solution is to do this for now:
```
        KeyE: (ev) => this._showQuickBar(ev),
        e: (ev) => this._showQuickBar(ev),
        KeyC: (ev) => this._showQuickBar(ev, true),
        c: (ev) => this._showQuickBar(ev, true),
        KeyM: (ev) => this._createMyLink(ev),
        m: (ev) => this._createMyLink(ev),
```
This will possibly add additional keys but it will cover all posibilities of e,c and m

Reply:
What problem would we have by going back to using the key ? I could not find any issue with this, so pliz explain :D

Reply:
@Salamandar as I said - it doesn't work when you use non-latin keyboards. French still uses the Latin character set more or less. Hebrew does not so it breaks. This way handles this but it adds a few more cases we don't intend to. However since this is non-latin it doesn't interfere with any functionality.
@bramkragten FYI. I think this can be accepted if @Salamandar makes this change.

Reply:
For my info, what keys would be added if we do it like this? Because it would restrict us in using more keys in the future then?

Reply:
@yosilevy Haaa alright, I think I understand the issue with non-latin : they do NOT have a "e" key *at all*, so non-latin users are expected to use the "e" from qwerty, thus your fix.

For me your solution isn't perfect at all. But at least it covers all the (known) use cases, so it's OK for me. I'm going to edit my pull request.

Reply:
@yosilevy I reverted the deletion of your code, and added comments explaining "what is what.